snapcraft.yaml

```diff
@@ -35,7 +31,7 @@ apps:
     desktop: usr/share/applications/com.github.xournalpp.xournalpp.desktop
     extensions: [gnome-3-38]
     environment:
-      PATH: $PATH:$SNAP/usr/bin:$SNAP/bin
+      PATH: $PATH:$SNAP/usr/bin:$SNAP/bin:$SNAP/usr/share/.TinyTeX/bin/x86_64-linux
```

Comment:
Can we make this path arch-independent?  Perhaps move it during the override-build step to be simply $SNAP/usr/share/.TinyTeX/bin/

Reply:
The trouble is that this directory contains relative links, which need to be regenerated when the directory is moved.

Reply:
Ok, perhaps just add an add both paths, it would be harmless

PATH: $PATH:$SNAP/usr/bin:$SNAP/bin:$SNAP/usr/share/.TinyTeX/bin/x86_64-linux:$SNAP/usr/share/.TinyTeX/bin/aarch64-linux

Reply:
I have also added the aarch64 path for Lua, so that the lgi module would be found on aarch64 as well.